simtools/data_model/model_data_writer.py

```diff
@@ -11,6 +16,23 @@
 __all__ = ["ModelDataWriter"]
 
 
+class JsonNumpyEncoder(json.JSONEncoder):
+    """Convert numpy to python types as accepted by json.dump."""
```

Comment:
I know this was in the code before, but I googled it and there seems to be a better way to do it, at least for the python types (using the `.item()` method). Please check the best answer here: 
[https://stackoverflow.com/questions/9452775/converting-numpy-dtypes-to-native-python-types](https://stackoverflow.com/questions/9452775/converting-numpy-dtypes-to-native-python-types)

Reply:
I disagree. Creating an encoder itself is not clearer than a simple one-line method. But as this is not necessarily part of the PR, I will leave it as it is then.

Reply:
To explain why I think the current way is better: `if isinstance(o, np.floating):` is understandable without knowing even python. `type(np.float64(0).item()) ` requires knowledge about the internals of  the numpy float64 class - I would have to look it up what it means. The encodes is required by the json writer.